docs/development/core/development-functional-tests.asciidoc

```diff
@@ -25,7 +25,10 @@ There are three ways to run the tests depending on your goals:
 *** can be reused for multiple executions of the tests, thereby saving some time when re-running tests
 *** automatically restarts the Kibana server when relevant changes are detected
 ** `node scripts/functional_test_runner`
-*** runs the tests against Kibana & Elasticsearch servers that were started by `node scripts/functional_tests_server`
+*** runs the UI tests against Kibana & Elasticsearch servers that were started by `node scripts/functional_tests_server`
```

Comment:
not accurate, runs all the tests, unless you pass in a single config for just UI tests.

Reply:
So can you run the API tests with just `node scripts/functional_test_runner`?

Reply:
see (comment)[https://github.com/elastic/kibana/pull/21738#discussion_r208352632]

Reply:
oh, runner! my bad. so... if you want to just start the functional test runner alone, because you already have the servers started (kibana and es), then you would do `node scripts/functional_test_runner --config path/to/config` (in this case the api tests config). again, this is explained [here](https://github.com/elastic/kibana/tree/master/scripts#functional-test-scripts)

Reply:
@nreese does the above make sense?